gcc-4.8/build.sh

```diff
@@ -55,5 +57,8 @@ make -j"$CPU_COUNT"
 make install-strip
 rm "$PREFIX"/lib64
 
+#Fix libtool paths
+find "$PREFIX"  -name '*.la' -print0 | xargs -0  sed -i 's%lib/../lib64%lib%g'
+
```

Comment:
Slight problem here:

On Linux, you can use `sed -i` without specifying a file extension for the backup files, but on OSX, the extension is required. (It's annoying.)

The following will work on both OS X and Linux.

```bash
find "$PREFIX" -name '*.la' -print0 | xargs -0  sed -i.backup 's%lib/../lib64%lib%g'
find "$PREFIX" -name '*la.backup' -print0 | xargs -0  rm -f
```